server/controllers/documentController.js

```diff
@@ -2,181 +2,196 @@ import db from '../models';
 
 
 const DocumentController = {
-  createNewDocument(req, res) {
-    db.documents
-      .create({
-        title: req.body.title,
-        content: req.body.content,
-        access: req.body.access || 'public',
-        userId: req.decodedToken.userId,
-        userRoleId: req.decodedToken.roleId
-      })
-      .then((document) => {
-        res.status(201).send({
-          message: 'Document created successfully',
-          data: document
-        });
-      })
-      .catch((err) => {
-        res.status(400).send({
-          message: 'There was an error while creating the document', err
-        });
+  createNewDocument(request, response) {
+    if (!request.body.title || !request.body.content) {
+      return response.status(400).send({
+        message:
+        'The title or the content of the document is empty'
```

Comment:
This message is wrong for the validation your doing. if I send an empty string in the title and content this validation will fail.